src/usdb_syncer/json.py

```diff
@@ -0,0 +1,114 @@
+"""Generates a JSON from the passed song list."""
+
+from __future__ import annotations
+
+import datetime
+import json
+from json import JSONEncoder
+from pathlib import Path
+from typing import Any, Iterable
+
+import attrs
+
+from usdb_syncer import SongId
+from usdb_syncer.logger import get_logger
+from usdb_syncer.resource_dl import _url_from_resource
```

Comment:
I'm surprised pylint doesn't complain about access of a private variable here.
But you can just mark it public by removing the leading underscore and potentially moving it to utils.py.

Reply:
oh true, this is still a dirty hack. Is there a public alternative - or should I make this function public?

Reply:
done in 61e57da